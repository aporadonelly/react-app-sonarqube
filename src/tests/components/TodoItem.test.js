import React from "react";
import "jest-enzyme";
import { configure, shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import TodoItem from "../../components/TodoItem";

configure({ adapter: new Adapter() });

describe("TodoItem", () => {
  it("should display todoitem with no error", () => {
    const wrapper = shallow(<TodoItem />);
    expect(wrapper).toMatchSnapshot();
  });
});
