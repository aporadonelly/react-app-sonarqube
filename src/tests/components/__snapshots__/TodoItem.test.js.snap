// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TodoItem should display todoitem with no error 1`] = `
<div
  className="todo-item"
>
  <h2 />
</div>
`;
